composer.json

```diff
@@ -18,13 +18,13 @@
     ],
 
     "require": {
-        "php": ">=5.3.1",
-        "behat/mink":  "~1.7",
-        "symfony/phpunit-bridge": "^4.0"
+        "php": ">=5.4",
+        "behat/mink":  "~1.7"
     },
-
     "require-dev": {
-        "symfony/http-kernel": "^2.7 || ^3.2 || ^4.0"
+        "symfony/http-kernel": "^2.7 || ^3.2 || ^4.0",
+        "phpunit/phpunit": ">=4.8,<8.0",
```

Comment:
if we decide to install phpunit through composer rather than through the phpunit-bridge `simple-phpunit` executable, it should be in `require` (for the same reason).

Reply:
reverted this part